public/js/utils/redux/middleware/log.js

```diff
@@ -9,7 +9,7 @@
  */
 function log({ dispatch, getState }) {
   return next => action => {
-    console.log("[DISPATCH]", JSON.stringify(action, null, 2));
+    console.log(`[DISPATCH ${action.type}]`, action);
```

Comment:
The pause action is a problem, yeah. Maybe we can log both the action object and the stringified version of it, and cap the stringified version to 1000 chars or something. That way we would ensure that you can always scroll through actions at a certain rate, but have the ability to interactively dig into the action if you want.